test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1181,10 +1151,9 @@ def test_fairshare(self):
         jid3 = self.submit_jobs(1, user=TEST_USER3)
 
         # Look at the job equivalence classes again
-        m = self.scheduler.log_match("Number of job equivalence classes: 1",
-                                     max_attempts=10,
-                                     starttime=int(self.t) - 1)
-        self.assertTrue(m)
+        self.scheduler.log_match("Number of job equivalence classes: 1",
+                                 max_attempts=10,
+                                 starttime=int(self.t) - 1)
```

Comment:
Why the -1?  This can be dangerous depending on the length of tests.  You might match a previous cycle's log message.  Also, some times you int(self.t) in the log_match() and other times it is int()'d at assignment time.  I'd like to see this consistent.